lasagne/tests/layers/test_conv.py

```diff
@@ -190,3 +224,227 @@ def test_with_nones(self, Conv2DImpl, DummyInputLayer,
 
         except NotImplementedError:
             pytest.skip()
+
+    def test_init_none_nonlinearity_bias(self, Conv2DImpl, DummyInputLayer):
+        input_layer = DummyInputLayer((1, 2, 3, 3))
+        layer = Conv2DImpl(input_layer, num_filters=16, filter_size=(3, 3),
+                           nonlinearity=None, b=None)
+        assert layer.nonlinearity == lasagne.nonlinearities.identity
+        assert layer.b is None
+
+    def test_invalid_border_mode(self, Conv2DImpl, DummyInputLayer):
+        input_layer = DummyInputLayer((1, 2, 3))
+        with pytest.raises(RuntimeError) as exc:
+            layer = Conv2DImpl(input_layer, num_filters=16, filter_size=(3, 3),
+                               border_mode='_nonexistent_mode')
+        assert "Invalid border mode" in exc.value.args[0]
+
+    def test_get_params(self, Conv2DImpl, DummyInputLayer):
+        input_layer = DummyInputLayer((128, 3, 32, 32))
+        layer = Conv2DImpl(input_layer, num_filters=16, filter_size=(3, 3))
+        assert layer.get_params() == [layer.W, layer.b]
+        assert layer.get_params(regularizable=False) == [layer.b]
+        assert layer.get_params(regularizable=True) == [layer.W]
+        assert layer.get_params(trainable=True) == [layer.W, layer.b]
+        assert layer.get_params(trainable=False) == []
+        assert layer.get_params(_nonexistent_tag=True) == []
+        assert layer.get_params(_nonexistent_tag=False) == [layer.W, layer.b]
+
+
+class TestConvOutputLength:
+    def test_invalid_border_mode(self):
+        from lasagne.layers.conv import conv_output_length
+        with pytest.raises(RuntimeError) as exc:
+            conv_output_length(5, 3, 1, border_mode='_nonexistent_mode')
+        assert "Invalid border mode" in exc.value.args[0]
```

Comment:
Yep. Which doesn't mean that it's not useful to add more tests for this function, I was just focusing on getting to 100% coverage.